pylintrc

```diff
@@ -1,314 +1,619 @@
-[MASTER]
+[MAIN]
```

Comment:
Are there any significant changes to the pylintrc or is it just an update to the current config schema?

Reply:
This is the only notable config option:
https://github.com/rebase-helper/rebase-helper/blob/8c06c0a33cd0de4a9657d5ca42345fdd8e0e4410/pylintrc#L540-L541

Reply:
The rest should be unchanged/default.